templates/patterns/gallery/gallery.ui_patterns.yml

```diff
@@ -0,0 +1,56 @@
+gallery:
+  label: "Gallery"
+  description: "Gallery of available media."
+  fields:
+    rows:
```

Comment:
Maybe we could call this `media`? As a gallery will showcase media items such as videos, images, etc.

Reply:
I was using the same naming as in the ecl since the parameters we pass to the twig template are indeed rows which can contain any media.

Reply:
After a conversation with @ademarco I change the structure of the pattern.